lib/handlers/ortc/edgeUtils.js

```diff
@@ -75,6 +75,11 @@ exports.mangleRtpParameters = function(rtpParameters)
 			delete codec.channels;
 		}
 
+		if (codec.mimeType && !codec.name)
+		{
+			codec.name = codec.mimeType.split('/')[1];
```

Comment:
We don't use `codec.name` but just `codec.mimeType` so this is not needed.

Reply:
Edge uses codec.name. If codec.name is not specified it returns InvalidAccessError.
See: https://draft.ortc.org/#rtcrtpcodecparameters*

RTCRtpCodecParameters.name seems to be required.

Reply:
You are completely right, my fault. Indeed adding `codec.name` is required in `edgeUtils.mangleRtpParameters()`, otherwise `this._handler.receive({ id, kind, rtpParameters });` in `transport.consume()` will fail.

Let me add it.

Reply:
I've pushed to v3 branch. Can you confirm it works before I release 3.2.4, please?

Thanks a lot.

Reply:
The v3 branch now works. Thanks.

Reply:
Great. Released in mediasoup-client 3.2.4.

Thanks!